packages/core/addon/templates/components/visualization-config/line-chart.hbs

```diff
@@ -1,12 +1,23 @@
 {{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 <span class="line-chart-config__header">Chart Style Options</span>
 <div class="line-chart-config__style-options">
+  {{#if displayStackOption}}
+    <div class="line-chart-config__stacked-opt">
```

Comment:
Can we move this toggle under the select with the other toggle?

Reply:
done
<img width="291" alt="Screen Shot 2019-07-03 at 11 47 05 AM" src="https://user-images.githubusercontent.com/13946669/60617300-4e859080-9d88-11e9-96f9-c0b34f62938c.png">